test/functional/fixtures/runner/uncaught-js-errors/handler-returns-true/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Page</title>
+    <script type="text/javascript">
+        window.onerror = function () {
+            return true;
+        };
+        throw  new Error('%TC_TEST_ERR%');

```

Comment:
We throw this error (`%TC_TEST_ERR%`) to check is it contained in the test report error when the test is failed. But now you check only the [error message prefix](https://github.com/georgiy-abbasov/testcafe-phoenix/pull/4/files#diff-7007dd3407786d8a738ebd53f27c271fR2)